flink-doris-connector/src/main/java/org/apache/doris/flink/source/DorisSource.java

```diff
@@ -95,6 +100,15 @@ public SourceReader<OUT, DorisSourceSplit> createReader(SourceReaderContext read
     public SplitEnumerator<DorisSourceSplit, PendingSplitsCheckpoint> createEnumerator(
             SplitEnumeratorContext<DorisSourceSplit> context) throws Exception {
         List<DorisSourceSplit> dorisSourceSplits = new ArrayList<>();
+        if (!resolvedFilterQuery.isEmpty()) {
```

Comment:
Is it possible that there is a NPE here?

Reply:
I will set an initial value